resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -30,7 +31,8 @@ object DbReader {
            WHERE
            (
              (pd1.status="Deleted" AND pd1.destroyedDate > now() - INTERVAL 30 DAY) OR
-             (pd1.status="Deleting" AND pd1.`dateAccessed` < now() - INTERVAL 30 MINUTE)
+             (pd1.status="Deleting" AND pd1.`dateAccessed` < now() - INTERVAL 30 MINUTE) OR
+             (pd1.destroyedDate == ${leonardoDummyDate})
```

Comment:
Would that include other PD status like RUNNING?

Reply:
oops...fixing